content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -188,9 +61,7 @@ export KF_NAME=<your choice of name for the Kubeflow deployment>
 export BASE_DIR=<path to a base directory>
 export KF_DIR=${BASE_DIR}/${KF_NAME}
 
-# Specify credentials for the default user.
-export KUBEFLOW_USER_EMAIL="admin@kubeflow.org"
-export KUBEFLOW_PASSWORD="12341234"
+# Credentials for the default user are admin@kubeflow.org:12341234
```

Comment:
Is it OK to put the credentials out in the open on the website like this? Especially as the username includes `kubeflow.org` it draws perhaps unwanted attention.

Reply:
The username and password are declared inside the KfDef file.
I will add a comment pointing the user to them.